.gitignore

```diff
@@ -2,5 +2,7 @@
 .test
 node_modules
 npm-debug.log*
+Consus-Client-*
+consus-logo.ico
```

Comment:
I don't see why we should ignore this file. I would think we would want all of our app icons in `assets/icons/` or something.